lib/utils/middlewares/__tests__/context_middleware.test.ts

```diff
@@ -11,21 +11,11 @@ describe("ContextMiddleware", () => {
   it("should extract the context from the request", async () => {
     const middleware = ContextMiddleware<ITestBindings>();
     const response = await middleware({
-      context: {
-        bindings: {
-          test: "test",
-        },
+      app: {
+        get: (s) => ({ context: {} }),
       },
     } as any);
-
     expect(response.isRight).toBeTruthy();
-    if (response.isRight) {
-      expect(response.right).toEqual({
```

Comment:
you should test that `response.right` is equal to the context at row 15, perhaps by returning a non-empty object there and testing for equality.